integration-tests/moonwall.config.json

```diff
@@ -0,0 +1,129 @@
+{
+   "label": "moonwall_config",
+   "defaultTestTimeout": 120000,
+   "scriptsDir": "scripts/",
+   "environments": [
+      {
+         "name": "zombienet_zeitgeist_upgrade",
+         "testFileDir": [
+            "tests/rt-upgrade-zombienet"
+         ],
+         "runScripts": [
+            "build-zeitgeist-spec.sh",
+            "download-polkadot.sh"
+         ],
```

Comment:
I think a script that builds zeitgeist in release mode should be contained within that list.

Reply:
The reason why I didn't do this is because the CI should only build the node once (for all tests), so that we can save CI runner resources and ultimately fasten the CI process.

Reply:
I think the we should fix that, Moonwall errors if the node is not build. You can check in the build script whether the script is in an CI environment by verifying the state of the [`GITHUB_ACTIONS`](https://docs.github.com/en/actions/learn-github-actions/variables) environment variable (`=true` if in CI).

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1192/commits/9e9a268d80005e27f136aeddb2335c81787eaaac

Reply:
wait it's not ready. the script needs to be executed from the root folder..

Reply:
d6585ded8d53f27d68d8cb6de254ba23d1e49664

Reply:
I will watch the CI running and if everything works, I will push another commit to remove the CI trigger for this branch and then I will request your review again.

Reply:
Unfortunately I get a `No space left on device` CI :-( So I rely on your advanced review.

Reply:
Good to know. I'll add the job to the self-hosted section once the new servers are ready.